### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by JesusGarce22**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by JesusGarce22 — COMMENTED** [review]
Chaged text

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (JesusGarce22) [inline index.html:50]: Add text

Comment (JesusGarces02) [inline index.html:50]: nice

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (JesusGarces02) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play againss 🧑‍💻 🤖!</h2>
```

**Pull request merged**